### ISSUE TITLE
grouped tabs thumbnail mode

### ISSUE THREAD

**Issue by Zerogaku** [mbnuqw/sidebery#1634]
### Description

On the grouped tabs page, there appears to be thumbnails for each webpage, can this be a more higlighted feature? As I often find myself lost in a sea of untitled web pages, having a thumbnail mode (a larger preview of each page and without the overlay or title covering the thumbnail of each web page) would be super helpful.

![image](https://github.com/mbnuqw/sidebery/assets/144664603/c0f70350-2e40-4d70-95cd-85704199478e)

**Comment by albino1** [comment]
Under `Settings` -> `Group Page` if you switch it from `Grid` to `List` it's a little better, but I agree, a third option for actual thumbnails would be much better.

**Comment by Michaelknubben** [comment]
I agree. I have a large monitor, and would like to be able to show text _and_ thumbnails
